### ISSUE TITLE
UIServer.stop() missing class org/slf4j/bridge/SLF4JBridgeHandler

### ISSUE THREAD

**Issue by printomi** [deeplearning4j/deeplearning4j#7274]
#### Issue Description

I tried to run UIServer.stop() in one of the new tests in `TestPlayUIMultiSession` in PR #7185, but it threw this exception:
```
java.lang.NoClassDefFoundError: org/slf4j/bridge/SLF4JBridgeHandler
	at play.api.Logger$.shutdown(Logger.scala:306)
	at play.core.server.Server$class.stop(Server.scala:69)
	at play.core.server.NettyServer.stop(NettyServer.scala:175)
	at play.server.Server.stop(Server.java:28)
	at org.deeplearning4j.ui.play.PlayUIServer.stop(PlayUIServer.java:422)
```
See [this gist](https://gist.github.com/printomi/c6999cfe57426d77630fb60b00ecf803) for the full exception stack trace.

This behavior can be also reproduced in [this example](https://github.com/deeplearning4j/dl4j-examples/blob/master/dl4j-examples/src/main/java/org/deeplearning4j/examples/userInterface/UIExample.java).

Do we miss a dependency in [pom.xml](https://github.com/deeplearning4j/deeplearning4j/blob/master/deeplearning4j/deeplearning4j-ui-parent/deeplearning4j-play/pom.xml)?

#### Version Information

* Deeplearning4j version: 1.0.0-SNAPSHOT, 1.0.0-beta3
* platform information (OS, etc): Windows 10, Java 8

#### Contributing

I guess, that this is a dependency issue. What do you think?

**Comment by AlexDBlack** [comment]
https://github.com/deeplearning4j/deeplearning4j/blob/master/deeplearning4j/deeplearning4j-ui-parent/deeplearning4j-play/pom.xml#L297-L301

Play Framework (2.4) has a hard requirement on logback.
This issue is likely due to the test scope for lombok here: https://github.com/deeplearning4j/deeplearning4j/blob/master/deeplearning4j/deeplearning4j-ui-parent/deeplearning4j-play/pom.xml#L297-L301

Not sure how that ended up with test scope; it should be compile scope. So set that scope to compile, or just remove it entirely and use the transient version of lombok.
I'll push up a fix for that some time this week, or feel free to send us a PR if you want.

**Comment by printomi** [comment]
I tried changing logback dependency scope from test to compile, but it did not help.
I rebuilt deeplearning4j-play with `clean install -Dmaven.test.skip=true`
I wrote and run this test case for the issue in TestPlayUI:
```java
@Test
    public void testUIServerStop() {
        UIServer uiServer = UIServer.getInstance(true, null);
        assertTrue(uiServer.isMultiSession());
        uiServer.stop();
        UIServer.getInstance(false, null);
        assertFalse(uiServer.isMultiSession());
    }
```

**Comment by printomi** [comment]
I have found a page about [jul-to-slf4](https://www.slf4j.org/legacy.html#jul-to-slf4j) and the JavaDoc of [SLF4JBridgeHandler](https://www.slf4j.org/api/org/slf4j/bridge/SLF4JBridgeHandler.html).

I also added jul-to-slf4j as a compile scope dependency, which eliminated the above mentioned exception.

However, I had to change `UIServer` and `PlayUIServer` to handle stopping and re-starting the server using the singleton pattern.

I have included these changes as a commit in PR #7185 , so that PR is now complete, I think.
